Samples/BackgroundBlur/BackgroundBlur/winmain.cpp

```diff
@@ -0,0 +1,572 @@
+// The main message window and UI for the streaming sample. 
+
+#include "Capture.h"
+#include "resource.h"
+#include "Helpers/common.h"
+#include "Helpers/trace.h"
+#include <shlobj.h>
+#include <Shlwapi.h>
+#include <powrprof.h>
+#include <KnownFolders.h>
+
+CaptureManager* g_pEngine = NULL;
+HPOWERNOTIFY    g_hPowerNotify = NULL;
+HPOWERNOTIFY    g_hPowerNotifyMonitor = NULL;
+SYSTEM_POWER_CAPABILITIES   g_pwrCaps{};
+bool            g_fSleepState = false;
+
+INT_PTR CALLBACK ChooseDeviceDlgProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam);
+
+#include <filesystem>
+#include <shlobj.h>
+
+static std::wstring GetLatestWinPixGpuCapturerPath_Cpp17()
```

Comment:
>static [](http://example.com/codeflow?start=0&length=6)

Break DLL effect into different folder from the main code so it can be reused independently